SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -2347,6 +2347,10 @@ void ActorSystem::SendCreateEntityRequest(USpatialActorChannel& ActorChannel, ui
 			OnEntityCreated(Op, SpanId);
 		});
 
+		if (ensure(EntityId != SpatialConstants::INVALID_ENTITY_ID))
+		{
+			CreateEntityRequestsInFlight.Add(EntityId);
+		}
 		CreateEntityRequestIdToActorChannel.Emplace(CreateEntityRequestId, MakeWeakObjectPtr(&ActorChannel));
```

Comment:
`CreateEntityRequestIdToActorChannel` actually seems redundant now, as we can get the actor from the entity id, and we don't need the actor channel at all

Reply:
But that's not a cleanup required now